fhir-persistence-schema/src/main/java/org/linuxforhealth/fhir/schema/app/util/CommonUtil.java

```diff
@@ -62,11 +62,10 @@ public static void configureLogger() {
     public static void logClasspath(Logger logger) {
         if (logger.isLoggable(Level.FINE)) {
             logger.fine("CLASSPATH: ");
-            ClassLoader cl = ClassLoader.getSystemClassLoader();
-            URL[] classpath = ((URLClassLoader) cl).getURLs();
-            for (URL u : classpath) {
-                logger.fine("  " + u.getFile());
-            }
+            String[] classPaths = System.getProperty("java.class.path").split(File.pathSeparator);
```

Comment:
is `System.getProperty("java.class.path")` a reliable way to get the classPath string?

is it always delimited by File.pathSeparator?  I might have expected ":" for *nix, and ";" for windows ... is the `java.class.path` documented somewhere?

Reply:
why not just log the result of `System.getProperty("java.class.path")` directly?

Reply:
Yes. You are right. I will edit the code.